### ISSUE TITLE
Fix test cases for the contentType related topic

### ISSUE THREAD

**Issue by derberg** [asyncapi/tck#27]
This issue is related to this PR https://github.com/asyncapi/tck/pull/26 and this followup issue https://github.com/asyncapi/asyncapi/issues/421.

tl;dr the spec is not clear on the fact that `contentType` or `defaultContentType` are mandatory but also does not specify what would be default in case they are not provided. It will probably by `application/json` but still, this is not stated. This is not a bug but rather an improvement to the spec so change will not be published quickly

Now we have these test cases:
- [tests/asyncapi-2.0/Message Trait Object/invalid-missing-content-type.yaml](https://github.com/asyncapi/tck/blob/master/tests/asyncapi-2.0/Message%20Trait%20Object/invalid-missing-content-type.yaml)
- [tests/asyncapi-2.0/Message Object/invalid-missing-contentType.yaml](https://github.com/asyncapi/tck/blob/master/tests/asyncapi-2.0/Message%20Object/invalid-missing-contentType.yaml)
- [tests/asyncapi-2.0/Message Object/valid-using-defaultContentType.yaml](https://github.com/asyncapi/tck/blob/master/tests/asyncapi-2.0/Message%20Object/valid-using-defaultContentType.yaml)
- [tests/asyncapi-2.0/Message Trait Object/valid-using-defaultContentType.yaml](https://github.com/asyncapi/tck/blob/master/tests/asyncapi-2.0/Message%20Trait%20Object/valid-using-defaultContentType.yaml)

The easiest would be in my opinion to mark those that are invalid as valid for now and probably keep this issue open to look into the topic again once we release newer versions of the spec

**Comment by derberg** [comment]
@postatum what do you think? I see 👍 up from @fmvilas. I can open a PR right away

**Comment by postatum** [comment]
> @postatum what do you think? I see +1 up from @fmvilas. I can open a PR right away

I'm ok with the changes you suggest. Would appreciate the PR! :+1:

**Comment by derberg** [comment]
@postatum it is out https://github.com/asyncapi/tck/pull/29/files
